hack/e2e-common.sh

```diff
@@ -54,7 +57,12 @@ function cluster_create {
 
 # $1 cluster
 function cluster_kind_load {
-	cluster_kind_load_image "$1" "$E2E_TEST_IMAGE"
+	e2e_test_sleep_image_without_sha=${E2E_TEST_IMAGE%%@*}
+	docker tag $E2E_TEST_IMAGE "$e2e_test_sleep_image_without_sha"
+    
+    # For more information https://github.com/kubernetes-sigs/kind/issues/2394#issuecomment-888713831.
+	# Manually create tag for image with digest which is already pulled already pulled.
+	cluster_kind_load_image "$1" "${e2e_test_sleep_image_without_sha}"
```

Comment:
```suggestion
	e2e_test_sleep_image_without_sha=${E2E_TEST_IMAGE%%@*}
	# We can load image by a digest but we cannot reference it by the digest that we pulled.
	# For more information https://github.com/kubernetes-sigs/kind/issues/2394#issuecomment-888713831.
	# Manually create tag for image with digest which is already pulled
	docker tag $E2E_TEST_IMAGE "$e2e_test_sleep_image_without_sha"
	cluster_kind_load_image "$1" "${e2e_test_sleep_image_without_sha}"
```

I would say like this to have more context.

Reply:
```suggestion
	e2e_test_sleep_image_without_sha=${E2E_TEST_IMAGE%%@*}
	# Manually create tag for image with digest which is already pulled.
	docker tag $E2E_TEST_IMAGE "$e2e_test_sleep_image_without_sha"
	# We can load image by a digest but we cannot reference it by the digest that we pulled.
	# For more information https://github.com/kubernetes-sigs/kind/issues/2394#issuecomment-888713831.
	cluster_kind_load_image "$1" "${e2e_test_sleep_image_without_sha}"
```

Or like this. But generally LGTM.